src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -59,29 +74,68 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     [messages, deleteMessage]
   );
 
+  const dropdownOptions = useMemo(
+    () =>
+      OPTIONS.map((option) => (
+        <Dropdown.Item
+          key={option.name}
+          onClick={() => setActiveOption(option)}
+        >
+          {option.name}
+          <FontAwesomeIcon icon={option.icon} />
+        </Dropdown.Item>
+      )),
+    []
+  );
+
+  const showPoll = useMemo(() => activeOption?.name !== OPTIONS[0].name, [
+    activeOption,
+  ]);
```

Comment:
What's this `OPTIONS[0].name` intending to do? I assume checking if the active option is the poll option. Once the code is refactored as @mike-lvov suggested in https://github.com/sparkletown/sparkle/pull/1343/files?diff=unified&w=1#r622457060 this should become a lot cleaner/clearer.

Reply:
Also, no need to `useMemo` a basic logical comparison like this. It runs quick enough that it's not worth the overhead of memoing

Reply:
fixed in b1c8ac0